Trend Micro/deep-security/ingest/parser.yml

```diff
@@ -88,7 +90,7 @@ stages:
           "75": "High"
           "100": "Very High"
         mapping:
-          parsed_event.message.cs5: event.risk_score_norm
+          event.severity: event.risk_score_norm
```

Comment:
I wonder if you don't have to change the mapping with integer on the left side.

Reply:
Applied ! =)

Reply:
I used the field `risk_score_norm` which seems more appropriate